libs/parallelism/algorithms/include/hpx/parallel/algorithms/is_sorted.hpp

```diff
@@ -42,17 +257,19 @@ namespace hpx { namespace parallel { inline namespace v1 {
             {
             }
 
-            template <typename ExPolicy, typename Pred>
+            template <typename ExPolicy, typename Pred, typename Proj>
             static bool sequential(
-                ExPolicy, FwdIter first, FwdIter last, Pred&& pred)
+                ExPolicy, FwdIter first, FwdIter last, Pred&& pred, Proj&& proj)
```

Comment:
Do you still plan to support iterator/sentinel pairs?

Reply:
Yes, very good catch here as well! I just forgot to update these.

Reply:
I've updated the base implementation to support this now as well.